config/v1/0000_10_config-operator_01_node.crd.yaml

```diff
@@ -43,6 +43,9 @@ spec:
                     - v1
                     - v2
                     - ""
+                eventedPleg:
+                  description: EventedPleg enables the corresponding flags both in the Kubelet and the CRI-O to support the Evented PLEG feature
```

Comment:
EventedPLEG enables event based PLEG between kubelet and CRI-O. (I would skip the description of how it does it).

Reply:
Updated.